va/va_vpp.h

```diff
@@ -1447,6 +1449,81 @@ typedef struct _VAProcFilterCap3DLUT {
     uint32_t            va_reserved[VA_PADDING_HIGH];
 } VAProcFilterCap3DLUT;
 
+/** @name 1DLUT Channel Layout and Mapping */
+/**@{*/
+/** \brief 1DLUT Channel Layout is unknown. */
+#define VA_1DLUT_CHANNEL_UNKNOWN              0x00000000
+/** \brief 1DLUT Channel Layout is R, G, B, the default layout. Map RGB to RGB. */
+#define VA_1DLUT_CHANNEL_RGB_RGB              0x00000001
+/** \brief 1DLUT Channel Layout is Y, U, V. Map YUV to YUV. */
+#define VA_1DLUT_CHANNEL_YUV_YUV              0x00000002
+/**@}*/
+
+/**
+  * \brief 1DLUT filter parametrization.
+  *
+  * 1DLUT (One Dimensional Look Up Table) has long been utilized in image processing such as monitor gamma table
+  * which typically leverages three 1DLUTs, one for each color channel. 1DLUT gives an output value for each of
+  * a range of index value.
+  *
+  * This buffer defines 1DLUT attributes and memory layout. The typical 1DLUT (y = f(x)) has fixed number(lut_size)
+  * entries which includes input value (x) and corrected/output value(y). For example, one dimensional identity LUT
+  * (lut_size = 4, bit_depth = 8), the index of entry is {0, 1, 2, 3}, input value (x) of the entry is {0, 84, 168, 255},
+  * corrected value (y) of the entry is {0, 84, 168, 255}. The lut_buffer points to the start of entry. The memory layout
+  * of lut buffer 1DLUT[lut_size][Entry]is as the below.
+  *     for (index = 0; index < lut_size; index++) {
+  *         1DLUT[index] = [Input_Value][Corrected_R/Y][Corrected_G/U][Corrected_B/V];
+  *     }
+  * The input value(Input_Value) can be any value of the range, for example, 8 bit image, it could be anyone in [0, 255],
+  * but must be in ascending order.
+  *
+  * 1DLUT usually does not change the format or color space of image. But when there are 1DLUT->3DLUT in the processing
+  * pipeline for a nonuniform lattice sampling, 3DLUT may change the format and color space, in this case, API user needs
+  * to set the correct 1DLUT/3DLUT mapping mode(1DLUT output should be same as 3DLUT input), implicit format conversion
+  * may be added automatically in the pipeline. For example, P010->1DLUT->3DLUT, if 3DLUT mapping mode is VA_3DLUT_CHANNEL_RGB_RGB,
+  * 1DLUT mapping mode should be same VA_1DLUT_CHANNEL_RGB_RGB, one format conversion could be added implicitly as the pipeline
+  * P010->RGB->1DLUT->3DLUT; if 3DLUT mapping mode is VA_3DLUT_CHANNEL_YUV_RGB, 1DLUT mapping mode
```

Comment:
Comparing `P010->1DLUT->3DLUT` `P010->RGB->1DLUT->3DLUT`, how should I understand the `implicit format conversion`? It seems to be a `yuv->rgb` conversion, right?